src/scripts/modules/games.js

```diff
@@ -0,0 +1,196 @@
+      card2.classList.remove('visible');
+      this.busy = false;
+    }, 1000);
+  }
+
+  canFlipCard(card) {
+    return !this.busy && !this.matchedCards.includes(card) && card !== this.cardToCheck;
+  }
+}
+
+const printGame = (arr) => {
+  const main = document.querySelector('main');
+  main.innerHTML = `
+  <section class="game">
+  <h2> Mix Or Match </h2>
+  <div class="overlay-text visible">
+    Click to Start
+  </div>
+  <div id="game-over-text" class="overlay-text">
+    GAME OVER
+    <span class="overlay-text-small">Click to Restart</span>
+  </div>
+  <div id="victory-text" class="overlay-text">
+    VICTORY
+    <span class="overlay-text-small">Click to Restart</span>
+  </div>
+  <div class="game-info">
+  <div class="game-info-time">
+  Time <span id="time-remaining">100</span>
+  </div>
+  <div class="game-info-flips">
+  Flips <span id="flips">0</span>
+  </div>
+  </div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  <div class="game-card"></div>
+  </section>
+  `;
+  const cards = [...document.querySelectorAll('.game-card')];
+  cards.forEach((card, index) => {
+    const logo = document.createElement('img');
+    const pokemon = document.createElement('img');
+    const front = document.createElement('div');
+    const back = document.createElement('div');
+    const cardWrapper = document.createElement('div');
+    logo.className = 'game-card-back-logo';
+    logo.src = PokeLogo;
+    pokemon.className = 'game-card-front-pokemon';
+    if (index > 7) {
+      pokemon.src = arr[index - 8].sprites.other.dream_world.front_default;
+    } else {
+      pokemon.src = arr[index].sprites.other.dream_world.front_default;
+    }
+    front.className = 'game-card-front';
+    back.className = 'game-card-back';
+    front.appendChild(pokemon);
+    back.appendChild(logo);
+    cardWrapper.className = 'game-card-wrapper';
+    cardWrapper.appendChild(front);
+    cardWrapper.appendChild(back);
+    card.appendChild(cardWrapper);
+  });
+  return cards;
+};
+
+const preGame = (cards) => {
+  const overlays = [...document.querySelectorAll('.overlay-text')];
+  const game = new PokemonGame(100, cards);
+  overlays.forEach((overlay) => {
+    overlay.addEventListener('click', () => {
+      overlay.classList.remove('visible');
+      game.startGame();
+    });
+  });
+  cards.forEach((card) => {
+    card.addEventListener('click', () => {
+      game.flipCard(card);
+    });
+  });
+};
+
+export default (arr) => {
+  const cards = printGame(arr);
+  preGame(cards);
+};
```

Comment:
It would be nice to add a function here that allows the player to see the cards before they're hidden. That way we would allow them to win easily 💯

Reply:
Hey @JohnFTitor! This is the first time I'm reading this comment, sorry about that. I think this one, in particular, is not something I would like to change because I actually like that you have to flip some cards before you understand what you are looking for. Honestly, I wanted to use random images from the first 200 pokemon so we could have a harder game but I didn't have the time last week. What do you think?

Reply:
Hello @carloshs1994 
Don't worry. I see your point. Maybe I didn't express myself correctly.
The game should be like that of course. The user needs to flip cards in order to match them. 

Question. Is this a memory game?
I thought it was a memory game because in that case, users need to first have a short, quick glance, to see all the cards flipped, before they hide again so the game starts. Because the way it's implemented right now makes you have to be wrong first in order to match them all.

If it was not intended as a memory game, then my bad. Please let me know which is the implementation you want 💯 
I agree. It would be a good idea. You can use this function

https://github.com/carloshs1994/choose-your-pokemon/blob/679f1e12abdc905d6c518c793d2c255dcf1b13b9/src/scripts/modules/APIhandling.js#L1-L11

To get the data within indexes. Please let me know if you have another idea. 
Cheers!